src/classes/Passage.ts

```diff
@@ -1,4 +1,12 @@
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
+<<<<<<< HEAD:src/Book.ts
```

Comment:
You left a merge conflict in the PR

Reply:
Ahh good catch idk why but I thought I could leave it in